mixins/test/label-mixin.test.js

```diff
@@ -86,38 +87,19 @@ describe('LabelledMixin', () => {
 
 	let elem;
 
-	beforeEach(async() => {
```

Comment:
I re-organized these a bit so that every test doesn't reuse the same giant fixture (which I was about to make bigger). The tests themselves haven't changed.